tests/network/expose.go

```diff
@@ -225,8 +280,10 @@ var _ = SIGDescribe("[rfe_id:253][crit:medium][vendor:cnv-qe@redhat.com][level:c
 				Expect(endpoint.Ports).To(ContainElement(k8sv1.EndpointPort{Name: "port-3", Port: 82, Protocol: "UDP"}))
 				Expect(endpoint.Ports).To(ContainElement(k8sv1.EndpointPort{Name: "port-4", Port: 1500, Protocol: "UDP"}))
 			},
-				table.Entry("[test_id:1533] over default IPv4 IP family", k8sv1.IPv4Protocol),
-				table.Entry("over IPv6 IP family", k8sv1.IPv6Protocol),
+				table.Entry("[test_id:1533] over default IPv4 IP family", ipv4),
+				table.Entry("over IPv6 IP family", ipv6),
+				table.Entry("over dual stack, primary ipv4", dualIPv4Primary),
+				table.Entry("over dual stack, primary ipv6", dualIPv6Primary),
```

Comment:
I do not see this test handling dual stack at all.

Reply:
The result for single stack and dual stack should be the same in this specific test.

Reply:
I am failing to understand the test then. It checks the ports, so how is that related to dualstack or not dual stack?
We configure something, but we do not check it... so what is the point exactly?

Reply:
@EdDev I think that this exactly the point - the extra configuration should not affect the functionality, thus the results should be the same as they were without the extra configuration.

Reply:
The fact that it asserts only the ports, makes this test agnostic to the family. It does not smell right even when single stack was tested.
But revisiting this later is reasonable, I don't want to insist on this thing. I find it unfitting e2e tests for sure.

Reply:
We're so used to ugly `if`s in our `DescribeTable` tests that if we don't have those if smells wrong:)
Here in this test we're checking the ports configuration is correct. The ports configuration per endpoint.subset. It it the same whether you have 1, 2, 10 or any other number of addresses in the subset. This test verifies it.

The test is not agnostic to the family. The service is created with the relevant families. The test verifies the port configuration is correct. In this case, the correct configuration is the same for all the families.